pyanaconda/nm.py

```diff
@@ -757,7 +757,11 @@ def nm_get_all_settings():
     connections = proxy.ListConnections()
     for con in connections:
         proxy = _get_proxy(object_path=con, interface_name="org.freedesktop.NetworkManager.Settings.Connection")
-        settings = proxy.GetSettings()
+        try:
+            settings = proxy.GetSettings()
+        except Exception as e:
+            log.debug("nm_get_all_settings: %s", e)
+            continue
```

Comment:
I think it would good to add a note why we are doing this - basically the content of the commit message. :)